### PR TITLE
Docs : Update guide for TIZEN RT IDE.

### PR DESCRIPTION
**Pull request by Hyunjun85**
Users can get and manage the certificate referring this.

HowToDevelopThingsWithTizenStudioForRT.md had devided as several  markdown files to reduce some confusions and it updated for Tizen RT IDE 2.2.

1. Update HowToDevelopThingsWithTizenStudioForRT.md.
2. Add manage-certificate.md. 
3. Add HowToManageDeviceModel.md.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/manage-certificate.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+# Managing SmartThings&trade; Certificates
+
+## Dependencies
+-   Ubuntu Only
```
Comment (JeongJunSik) [inline docs/manage-certificate.md]: Remove this.

File: docs/manage-certificate.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+To import an existing certificate:
+1.  In the SmartThings Certificate Manager, click **Import**.
+2. In the Certificate Import Dialog, enter the required information to import a certificate and click **OK**:
+
+    -   **Certificate Name**: Unique name for the certificate
+    -   **Private Key**: Private key file in the `.der` format
+    -   **Certificate**: Certificate file in the `.pem` format
+
+    ![Certificate Import Dialog](media/rt_cert_import.png)
```
Comment (sunghan-chang) [inline docs/manage-certificate.md]: You missed png file.

**Comment by sunghan-chang** [comment]
```
Add guide doc for managing certificate.
Update user guild for managing certification. 
```
Your guides are only for SmartThings. Let's use below:
```
Add guide doc for managing SmartThings certificate.
Update user guild for managing SmartThings certification. 
```

File: docs/manage-certificate.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+
+# Managing SmartThings&trade; Certificates
+
+## Dependencies
```
Comment (JeongJunSik) [inline docs/manage-certificate.md]: remove this.

File: docs/manage-certificate.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+-   Ubuntu Only
+
+SmartThings&trade; uses OpenSSL for security. To ensure that only trusted developers and manufactures can develop devices that communicate with the SmartThings Cloud, you must get certificates for devices to use the SmartThings&trade; service.
+
+In the Tizen RT IDE, you can perform various certificate-related tasks, such as creation, removal, import, and activation, by using the **SmartThings Certificate Manager**.
+
+**Figure: SmartThings Certificate Manager**
```
Comment (JeongJunSik) [inline docs/manage-certificate.md]: remove this

File: docs/manage-certificate.md (hunk 2 above)
Comment (JeongJunSik) [inline docs/manage-certificate.md]: Plz, Check this image

File: docs/manage-certificate.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+<a name="activate"></a>
+## Activating a Certificate
+
+To flash a project, you must have an active certificate. The Tizen RT IDE copies the activated certificate to the project during the flashing process.
+
+To activate a certificate:
+
+1.  In the SmartThings Certificate Manager, select the certificate you want to activate.
+2.  Click **Set Active**.
```
Comment (JeongJunSik) [inline docs/manage-certificate.md]: add image for this descripter

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/manage-certificate.md (hunk 2 above)
Comment (Hyunjun85) [reply docs/manage-certificate.md]: This is uploaded.

Comment (Hyunjun85) [reply docs/manage-certificate.md]: This is uploaded.

File: docs/manage-certificate.md (hunk 5 above)
Comment (Hyunjun85) [reply docs/manage-certificate.md]: We can see **Set Active" on top image.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by sunghan-chang** [comment]
This PR has 5 commits. Is it right?

File: docs/manage-certificate.md (hunk 3 above)
Comment (Hyunjun85) [reply docs/manage-certificate.md]: Has removed.

File: docs/manage-certificate.md (hunk 1 above)
Comment (Hyunjun85) [reply docs/manage-certificate.md]: Has removed.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request closed without merging**